<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutWidgetLed"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBoxMovie">
          <property name="title">
           <string>Movie</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="btnNewMovie">
                 <property name="text">
                  <string>New Movie</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSaveToFile">
                 <property name="text">
                  <string>Save To File</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnLoadFromFile">
                 <property name="text">
                  <string>Load From File</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btnPlayStopMovie">
               <property name="text">
                <string>Play Movie</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxFrame">
          <property name="title">
           <string>Frame Editor</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btnFrameClear">
               <property name="text">
                <string>Clear Frame</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFrameInsertBefore">
               <property name="text">
                <string>Insert Frame Before</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFrameInsertAfter">
               <property name="text">
                <string>Insert Frame After</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnFrameDelete">
               <property name="text">
                <string>Delete Frame</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnLoadFromFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnLoadFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveToFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnSaveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNewMovie</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnNewMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPlayStopMovie</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnPlayStopMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFrameClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnFrameClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFrameDelete</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnFrameDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFrameInsertBefore</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnFrameInsertBefore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnFrameInsertAfter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onBtnFrameInsertAfter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnLoadFromFile()</slot>
  <slot>onBtnSaveToFile()</slot>
  <slot>onBtnNewMovie()</slot>
  <slot>onBtnPlayStopMovie()</slot>
  <slot>onBtnFrameClear()</slot>
  <slot>onBtnFrameInsertBefore()</slot>
  <slot>onBtnFrameInsertAfter()</slot>
  <slot>onBtnFrameDelete()</slot>
 </slots>
</ui>
